File: ASSIGNMENT/Module 2.2/factorial.c
//WAP to print factorial of given number

#include<stdio.h>
int main()
{
    int fact = 1;
    int num ;
    printf("enter the number : ", num);
    scanf("%d", &num);

    for(int i = 1 ; i <= num ; i++)
    {
        fact = fact * i ;
    }
    printf("\nfactorial of %d is : %d", num , fact);
}

File: JavaScript/8-Function/Recursion/Factorial.js

function factorial() {
    let n = parseInt(document.getElementById("no").value);
    let fact = document.getElementById("ans");
    if (isNaN(n)) {
        fact.innerHTML = "Please enter a valid number";
    } else {
        function recursion(n) {
            if (n <= 1) {
                return 1;
            } else {
                return n * recursion(n - 1);
            }
        }
        fact.innerHTML = `Factorial of ${n} is: ${recursion(n)}`;
    }
}